config/groups.yaml

```diff
@@ -727,3 +727,8 @@
     - "2833207"
     - "2637174"
   id: 190178
+ - name: "The prototype"
+  members:
```

Comment:
This isn't valid YAML. See the other entries in the file for how to format this correctly.